lib/microsoft_kiota_serialization_json/json_serialization_writer.rb

```diff
@@ -174,8 +174,8 @@ def write_object_value(key, value)
       end
     end
 
-    def write_enum_value(key, values)
-      self.write_string_value(key, values.to_s)
+    def write_enum_value(key, value)
+      self.write_string_value(key, value)
```

Comment:
```suggestion
      self.write_string_value(key, value.to_s) unless value.nil?
```

What do you think about this alternative?